src/main/java/com/codedx/burp/ExportActionListener.java

```diff
@@ -0,0 +1,132 @@
+			if(report != null && report.exists()){
+				try{
+					HttpResponse response = sendData(report, getServer());
+					StatusLine responseLine = response.getStatusLine();
+					int responseCode = responseLine.getStatusCode();
+					if(responseCode == 202){
+						burpExtender.message("The report was successfully uploaded to Code Dx.", "Success");
+					} else if(responseCode == 400) {
+						burpExtender.error("An unexpected error occurred and the report could not be sent.\nThe server returned Error 400: Bad Request" + getResponseError(response));
+					} else if(responseCode == 403){
+						burpExtender.error("The report could not be sent. The server returned Error 403: Forbidden.\nThe API Key may be incorrect or have insufficient permissions for this project.");
+					} else if(responseCode == 404){
+						burpExtender.error("The report could not be sent. The server returned Error 404: Not Found.\nThe Server URL may be wrong or the project may no longer exist.");
+					} else if(responseCode == 415) {
+						burpExtender.error("An unexpected error occurred and the report could not be sent.\nThe server returned Error 400: Bad Request" + getResponseError(response));
+					} else {
+						burpExtender.error("An unexpected error occurred and the report could not be sent.\nThe response code is: " + responseLine);
+					}
+				} catch (IOException e1){
+					burpExtender.error("An unexpected error occurred and the report could not be sent.");
+				}
+				report.delete();
+			} else {
+				burpExtender.error("The report file could not be created.");
+			}
+		} else {
+			burpExtender.warn("There are no issues with the selected target.");
+		}
+	}
+	
+	private String getResponseError(HttpResponse response){
+		String msg = ".";
+		try (BufferedReader rd = new BufferedReader(new InputStreamReader(response.getEntity().getContent(), "UTF-8"))){
+			StringBuffer result = new StringBuffer();
+			String line = "";
+			while ((line = rd.readLine()) != null) {
+				result.append(line);
+			}
+			
+			JSONObject obj = new JSONObject(result.toString());
+			msg = " with the response: " + obj.getString("error");
+		} catch (JSONException | IOException e){}
+		return msg;
+	}
+	
+	private HttpResponse sendData(File data, String urlStr) throws IOException{
+		CloseableHttpClient client = burpExtender.getHttpClient();
+		HttpPost post = new HttpPost(urlStr);
+        post.setHeader("API-Key", burpExtender.getApiKey());
+        
+		MultipartEntityBuilder builder = MultipartEntityBuilder.create();
+		builder.setMode(HttpMultipartMode.BROWSER_COMPATIBLE);
+		builder.addPart("file", new FileBody(data));
+		
+		HttpEntity entity = builder.build();
+		post.setEntity(entity);
+
+	    HttpResponse response = client.execute(post);
+	    HttpEntity resEntity = response.getEntity();
+	    StatusLine responseCode = response.getStatusLine();
+	    
+	    if (resEntity != null) {
+	    	EntityUtils.consume(resEntity);
+	    }
+	    client.close();
+	    
+		return response;
+	}
```

Comment:
Indentation in this function looks messed up on github. Probably mixing tabs and spaces.